torch/autograd/grad_mode.py

```diff
@@ -57,7 +57,7 @@ class no_grad(_NoParamDecoratorContextManager):
         >>> z = doubler(x)
         >>> z.requires_grad
         False
-        >>> @torch.no_grad
+        >>> @torch.no_grad()
```

Comment:
@Chillee `@torch.no_grad` works fine. See: https://github.com/pytorch/pytorch/blob/d261687d5f56ac8148fab2567cf1fa6dd5264def/torch/utils/_contextlib.py#L120

I think this annotation is cleaner than with `()`

Reply:
Hmm, that being said it seems that all uses of these in torch put the brackets, I guess changing makes sense to make the docs consistent with the usage.